yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -866,23 +910,11 @@ object Client extends Logging {
    * Get the list of namenodes the user may access.
    */
   private[yarn] def getNameNodesToAccess(sparkConf: SparkConf): Set[Path] = {
```

Comment:
is this is single call, we should just delete this and replace it with the single line wherever its called.